torch/csrc/jit/fusion_compiler.cpp

```diff
@@ -798,7 +804,7 @@ void FusedKernel::launch_with_tensors(at::ArrayRef<at::Tensor> inputs, at::Array
   arguments.reserve(3 + flat_inputs_size + flat_outputs_size);
   auto addTensorInfoRaw = [&](TensorDesc & desc, void* data_ptr, at::IntList sizes, at::IntList strides) {
     size_t nDim = desc.nDim(); // NOTE: this is the compressed dim
```

Comment:
Maybe it's worth fixing `nDim` to return 0 instead? Scalars don't need any indexes, so the code should be easy to generalize, and wouldn't need all the conditionals above.